jaseci_kit/jaseci_kit/modules/fasttext/README.md

```diff
@@ -1,9 +1,501 @@
+        - ShareETA
+        - GetTrafficInformation
+        - ComparePlaces
+        - ShareCurrentLocation
+        fasttext validation started...
+        {
+        "success": true,
+        "report": [
+            "Model training Completed"
+        ]
+        }
+        ```
+### **4. Evaluation of the model effectiveness** </br>
+* Performing model effectiveness on `test.json` dataset
+
+    ```
+    Model testing Accuracy :  0.82
+    Model testing F1_Score :  0.76
+
+    Model classification Report
+
+                            precision    recall  f1-score   support
+
+            BookRestaurant       1.00      1.00      1.00        14
+             ComparePlaces       1.00      0.25      0.40         4
+             GetDirections       0.70      1.00      0.82         7
+           GetPlaceDetails       0.56      0.90      0.69        10
+     GetTrafficInformation       0.80      1.00      0.89         4
+                GetWeather       0.88      0.78      0.82         9
+               RequestRide       1.00      1.00      1.00         5
+               SearchPlace       0.00      0.00      0.00         6
+      ShareCurrentLocation       1.00      1.00      1.00         3
+                  ShareETA       1.00      1.00      1.00         4
+
+                  accuracy                           0.82        66
+                 macro avg       0.79      0.79      0.76        66
+              weighted avg       0.78      0.82      0.78        66
+    ```
+
+    **Sample Result Data**
+    ```
+    {
+        "I want a table for friday 8pm for 2 people at Katz's Delicatessen": [
+            {
+                "sentence": "I want a table for friday 8pm for 2 people at Katz's Delicatessen",
+                "intent": "BookRestaurant",
+                "probability": 0.9759947061538696
+            }
+        ],
+        "I want a table in a good japanese restaurant near Trump tower": [
+            {
+                "sentence": "I want a table in a good japanese restaurant near Trump tower",
+                "intent": "BookRestaurant",
+                "probability": 0.830787181854248
+            }
+        ],
+        "Book a table at a restaurant near Times Square for 2 people tomorrow night": [
+            {
+                "sentence": "Book a table at a restaurant near Times Square for 2 people tomorrow night",
+                "intent": "BookRestaurant",
+                "probability": 0.9866142272949219
+            }
+        ],
+        "Book a table for today's lunch at Eggy's Diner for 3 people": [
+            {
+                "sentence": "Book a table for today's lunch at Eggy's Diner for 3 people",
+                "intent": "BookRestaurant",
+                "probability": 0.9936538934707642
+            }
+        ]
+    }
+    ```
+
+### **5. Use the trained model to make predictions** </br>
+* Create new input data for prdiction stored in `test_dataset.json` file (can take any name) </br>
```

Comment:
change this 
in `test_dataset.json` file (can take any name)
to 
in  a file for example - `test_dataset.json`

Reply:
done